lib/lower/ASTBuilder.cpp

```diff
@@ -0,0 +1,872 @@
+//===-- lib/lower/ast-builder.cc ------------------------------------------===//
```

Comment:
The relative path name in the license header does not match the name of the file.